examples/typescript/package-lock.json

```diff
@@ -0,0 +1,11448 @@
+{
+  "name": "typescript",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
+      "dependencies": {
+        "jest": "^26.6.3",
+        "mocha": "^8.4.0"
+      }
+    },
+    "node_modules/@babel/code-frame": {
+      "version": "7.12.13",
+      "resolved": "http://artifactory.inf.las1.saucelabs.net:80/artifactory/api/npm/all-npm/@babel/code-frame/-/code-frame-7.12.13.tgz",
```

Comment:
Should regenerate the lockfile with the public registry.

Reply:
Thanks! I changed to the public registry.